js/src/pages/price-benchmark/index.js

```diff
@@ -13,6 +16,34 @@ import Banner from './banner';
 import './index.scss';
 
 const PriceBenchmark = () => {
+	const [ dataViewLoaded, setDataViewLoaded ] = useState();
+
+	useEffect( () => {
+		if ( dataViewLoaded === undefined ) {
+			const script = document.createElement( 'script' );
+			script.src = glaData.dataViewsScriptUrl;
```

Comment:
@ankitrox You can import `glaData` from `constants` and then access `dataViewsScriptUrl`
`import { glaData } from '~/constants';`